modules/source_navigation.py

```diff
@@ -131,11 +131,39 @@ async def navigate_to_source(self, source: dap.SourceLocation, move_cursor: bool
 			view.set_read_only(True)
 			view.set_scratch(True)
 		elif source.source.path:
-			view = await core.sublime_open_file_async(self.project.window, source.source.path, group=0)
+			if source.source.path.startswith("jdt:"):
```

Comment:
@daveleroy This is currently only a dirty hack as proof-of-concept. Would it be possible to allow adapters to do the overwrite the navigation or at least supply content for special URIs?

Reply:
@rwols Do you have an idea how I could wait for the view created by LSP? The view is needed to set the name of the view and call show_line.

Reply:
I "fixed" it by not using the LSP URI logic